### PR TITLE
[Logs] Update field descriptions for Logs

### PR DESCRIPTION
**Pull request by cjolowicz**
This update contains two changes related to Security Event Logs improvements:

* Synchronize field descriptions with Logshare templates

  - `gateway_http.md`
  - `gateway_network.md`
  - `http_requests.md`

* Rename "Firewall" to "Security" in `firewall-fields.md`

  - Rename file to `security-fields.md`
  - Rename "Firewall fields" to "Security fields" in the title, heading, and description
  - Rename `FirewallMatchesActions` to `SecurityActions` in the subheadings
  - Rename `FirewallMatchesSources` to `SecuritySources` in the subheadings
  - Redirect `/logs/reference/{firewall => security}-fields/` (301)

cc @pedrosousa 
cc @bharatnc

### CODE REVIEW

File: content/logs/reference/security-fields.md
Comment (pedrosousa) [inline content/logs/reference/security-fields.md:1]: We need to add a redirect for this renamed file.

File: content/logs/reference/log-fields/account/gateway_http.md [hunk 1]
```diff
@@ -31,7 +31,9 @@ The descriptions below detail the fields available for `gateway_http`.
 | FileInfo | Information about files detected within the HTTP request. | object |
 | HTTPHost | Content of the host header in the HTTP request. | string |
 | HTTPMethod | HTTP request method. | string |
+| HTTPStatusCode | HTTP status code gateway returned to the user. 0 if nothing was returned(e.g client disconnected) | int |
```
Comment (angelampcosta) [inline content/logs/reference/log-fields/account/gateway_http.md]:
```suggestion
| HTTPStatusCode | HTTP status code gateway returned to the user. Zero if nothing was returned (for example, client disconnected). | int |
```

File: content/logs/reference/log-fields/account/gateway_http.md [hunk 2]
```diff
@@ -40,7 +42,7 @@ The descriptions below detail the fields available for `gateway_http`.
 | SourceIP | Source ip of the request. | string |
 | SourcePort | Source port of the request. | string |
 | URL | HTTP request URL. | string |
-| UntrustedCertificateAction | Action taken when an untrusted origin certificate error occurs (e.g. expired certificate, mismatched common name, invalid certificate chain, signed by non-public CA). One of <em>none</em> \| <em>block</em> \| <em>passThrough</em>. | string |
+| UntrustedCertificateAction | Action taken when an untrusted origin certificate error occurs (e.g. expired certificate, mismatched common name, invalid certificate chain, signed by non-public CA). One of <em>none</em> \| <em>block</em> \| <em>error</em> \| <em>passThrough</em>. | string |
```
Comment (angelampcosta) [inline content/logs/reference/log-fields/account/gateway_http.md]:
```suggestion
| UntrustedCertificateAction | Action taken when an untrusted origin certificate error occurs (for example, expired certificate, mismatched common name, invalid certificate chain, signed by non-public CA). One of <em>none</em> \| <em>block</em> \| <em>error</em> \| <em>passThrough</em>. | string |
```

**Comment by angelampcosta** [comment]
@cjolowicz Just to double check, this PR will include https://github.com/cloudflare/cloudflare-docs/pull/8117?
cc @norwoodj

**Comment by cjolowicz** [comment]
@angelampcosta 

> Just to double check, this PR will include #8117?

Correct, it did. That PR has been merged, and I've rebased mine. Updated description to reflect.

File: content/logs/reference/security-fields.md
Comment (cjolowicz) [reply content/logs/reference/security-fields.md:1]: I've added the redirect in `content/_redirects`.

It seems the redirects don't work locally with `npm run dev` though? Haven't been able to verify it's working, but it looks like it should :crossed_fingers:

**Review by soheiokamoto — APPROVED** [review]

**Comment by soheiokamoto** [comment]
Changes on description seemed to be made manually in this PR, but not reflected or fixed in the internal source (`data/logshare-api`).
It will lose any manual changes unless it is updated at the source, i.e., https://github.com/cloudflare/cloudflare-docs/pull/8258

**Comment by soheiokamoto** [comment]
> Changes on description seemed to be made manually in this PR, but not reflected or fixed in the internal source (`data/logshare-api`). It will lose any manual changes unless it is updated at the source, i.e., #8258

Fixed in the internal source.

**Comment by cjolowicz** [comment]
Thanks @soheiokamoto and apologies. The PR was originally generated from logshare-api, but I committed suggested changes instead of porting them back into the internal source.

**Pull request merged**